Add Qwiklabs link to front page and fix formatting.

deeglaze opened the pull request:
Change to buttons screenshot https://imgur.com/a/ti6y9F7

Spacing in markdown is to keep the auto-formatter from concatenating lines.

KeithMoyer commented on index.html:
```diff
@@ -81,8 +81,9 @@ <h2>Protect your data and code in use and at rest</h2>
             <h2>Want to learn more?</h2>
-            <p>Get started with Asylo by running through our quickstart guide.</p>
-            <a class="btn btn-asylo" href="{{home}}/docs/guides/quickstart.html">GET STARTED</a>
+            <p>Get started with Asylo via our quickstart guide, or our free Qwiklabs exercises.</p>
```
nit: no comma

KeithMoyer replied:
Could this be split into three commits (one PR is fine), for the different unrelated items?
 - Formatting
 - Qwiklabs link
 - std::string

KeithMoyer commented on index.html:
```diff
@@ -81,8 +81,9 @@ <h2>Protect your data and code in use and at rest</h2>
             <h2>Want to learn more?</h2>
-            <p>Get started with Asylo by running through our quickstart guide.</p>
-            <a class="btn btn-asylo" href="{{home}}/docs/guides/quickstart.html">GET STARTED</a>
+            <p>Get started with Asylo via our quickstart guide, or our free Qwiklabs exercises.</p>
```
As far as I'm aware, Qwiklabs are not free (must pay for the associated GCP resources, to avoid abuse - e.g., cryptocurrency mining). There is code currently provided that removes the need to pay for those resources for our labs, but it expires 2019-04-01 (when C3 ends).

deeglaze replied:
Removed "free"

deeglaze committed:
Add Qwiklabs link to the main page

deeglaze committed:
Change string to std::string in guides

deeglaze committed:
Add spacing and reformat

The lack of spacing between yaml tags in Jekyll frontmatter
causes some issues with the markdown formatter we use. Add
spacing for these pages to cooperate.
